app/views/directives/notifications/notification-drawer-wrapper.html

```diff
@@ -0,0 +1,18 @@
+<pf-notification-drawer
+  drawer-hidden="$ctrl.drawerHidden"
+  allow-expand="$ctrl.allowExpand"
+  drawer-expanded="$ctrl.drawerExpanded"
+  drawer-title="{{$ctrl.drawerTitle}}"
+  show-clear-all="$ctrl.showClearAll"
+  show-mark-all-read="$ctrl.showMarkAllRead"
+  notification-groups="$ctrl.notificationGroups"
+  action-button-title="{{$ctrl.actionButtonTitle}}"
+  action-button-callback="$ctrl.actionButtonCallback"
+  heading-include="{{$ctrl.headingInclude}}"
+  subheading-include="{{$ctrl.subheadingInclude}}"
+  notification-body-include="{{$ctrl.notificationBodyInclude}}"
+  notification-footer-include="{{$ctrl.notificationFooterInclude}}"
```

Comment:
I don't see actionButtonTitle, actionButtonCallback , subHeadingInclude, or notificationFooterInclude defined in the controller. These can be omitted.